whitelabel_automator.egg-info/PKG-INFO

```diff
@@ -0,0 +1,11 @@
+Metadata-Version: 1.1
```

Comment:
the `egg-info` is autogenerated - could we cut this directory out? i can include it in the gitignore for the future.

Reply:
[same thing with removing the dist directory

Reply:
yeah we can add it to the gitignore! works for me

Reply:
sgtm! deleted theses file off master and added to the .gitignore.